.github/workflows/push.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           cache: maven
 
       - name: build with maven
-        run: mvn clean install
+        run: mvn clean install -Dgithub.repo=${{ github.repository }} -Dgit.sha.commit=${{ github.event.pull_request.head.sha }}
```

Comment:
I don't think we need to pass these properties here any longer as we're now using defaults.
Also, this ensures the `mvn clean install` works as it is.